nginx/datadog_checks/nginx/nginx.py

```diff
@@ -141,10 +152,6 @@ def check(self, instance):
                 func = funcs[metric_type]
                 func(name, value, tags)
 
-                # for vts and plus versions
```

Comment:
What's vts?

Reply:
A nginx module for virtual traffic (https://github.com/vozlt/nginx-module-vts)